### DISCUSSION TITLE
Gmail tiukentaa vaatimuksia massasähköpostien lähettäjille - vaatii muutoksia kirjastojen sähköpostiosoitteisiin

### DISCUSSION THREAD

**Discussion by AnneliO** [KohaSuomi/Koha#1052] [Tiedotteet]
Saimme viime viikon torstaina tiedon, että Google asettaa uusia vaatimuksia massasähköpostien lähettäjille varmistaakseen Gmailin luotettavuuden ja turvallisuuden vastaanottajille. Myös Yahoo on ottamassa vastaavat vaatimukset käyttöönsä. Muutokset on suunnattu erityisesti lähettäjille, jotka lähettävät Gmail-osoitteisiin yli 5000 viestiä päivässä. Koha-Suomen kimpoista lähetettiin esim. viime maanantaina yli 16 000 sähköpostiviestiä. Googlen muutos tulee voimaan 1.2.2024.

Uudet vaatimukset:
 
• SPF- ja DKIM-todennus: Jatkossa sähköpostin lähettäjien, jotka lähettävät Gmail-osoitteisiin yli 5 000 sähköpostia päivässä, on otettava käyttöön sekä SPF- että DKIM-todennus omalla verkkotunnuksellaan. SPF auttaa tarkistamaan, että viesti on lähetetty sallitusta lähteestä, kun taas DKIM varmistaa sähköpostin alkuperän ja eheyden.
 
• DMARC-tietue: Lähettäjien, jotka lähettävät päivässä yli 5 000 sähköpostia Gmailiin, on nyt asetettava verkkotunnukselleen DMARC (Domain-based Message Authentication, Reporting, and Conformance)-tietue, jonka vähimmäistaso on p=none. DMARC yhdistää SPF:n ja DKIM:n, ja se antaa verkkotunnuksen omistajalle kattavan kuvan sähköpostien todennuskäytännöistä.

Muutokset on nyt tehty Koha-Suomen verkkotunnukselle ja palvelimille. Jotta määritykset toimisivat oikein, pitää kaikissa kimpoissa muuttaa kirjastojen lähetysosoitteeksi noreply@koha-suomi.fi. Lähetysosoitteen muutos onnistuu Ylläpidossa Kirjastot-osiossa. Lähetysosoite on Sähköposti-kentässä. Jos on tarvetta, voi kyseisen kentän alapuolella olevaan Vastausosoite-kenttään laittaa kirjaston oikean osoitteen, jolloin jos asiakkaat vastaavat sähköpostiviestiin, ohjautuu viesti kyseisen kirjaston osoitteeseen. Muutos kannattaa tehdä mahdollisimman pian.

Lisäksi kannattaa huomioida nämä asiat:

* Jos viestipohjissa on käytetty kirjaston sähköpostin viestiin lisäävää "tägiä" branches.branchemail, kannattaa ne muuttaa vastaussähköpostiosoitteen "tägiksi" branches.branchreplyto.
* Jatkossa ilmoitukset siitä, että viestin lähettäminen asiakkaan osoitteeseen on epäonnistunut, ohjautuvat tuohon noreply@koha-suomi.fi -osoitteeseen, jota ei käytännössä ole olemassa eikä sitä pääse lukemaan. Eli jos kirjastoissa on ollut tapana käydä näitä ilmoituksia läpi, niin jatkossa niitä ei enää tule.
* Asiakkaita kannattaa jollain tavalla tiedottaa muutoksesta, koska muuttunut osoite voi huolestuttaa osaa asiakkaista. Varsinkin kun nykyään varoitellaan paljon oudoista osoitteista tulevista huijausviesteistä. 

Osalla kimpoista on jo ennestään käytössä lähettäjäosoitteena noreply@koha-suomi.fi, koska sillä korjattiin jo aiemmin ongelmia Gmailin kanssa. Kaikissa kimpoissa sitä ei ole tätä ennen otettu käyttöön, mutta nyt muutos on käytännössä pakollinen, jotta viestit menevät jatkossakin Gmailia käyttäville asiakkaille perille.

**Comment by PiaKusmin** [comment]
Lappi korjattu ja tiedotettu.
